aiohttp_jinja2/__init__.py

```diff
@@ -24,6 +24,8 @@ def setup(app, *args, app_key=APP_KEY, context_processors=(),
         env.globals.update(GLOBAL_HELPERS)
     if filters is not None:
         env.filters.update(filters)
+    if 'autoescape' not in kwargs:
+        env.autoescape = lambda _: True
```

Comment:
`env.autoescape = True` would be enough

Reply:
feel like it would've been easier to fix this yourself...

Reply:
1. You are asking for new feature.
2. I personally don't need it but the feature looks interesting, I'm ready for finding a time for review.
3. You are providing a PR -- awesome. The PR is not perfect. It requires a couple iterations before merging. You give up after first step.

Do you need the feature? If yes -- please finish the PR. The project is driven by OSS enthusiasts in spare time, personally  I busy on other (and more important for me) projects.

Reply:
To be clear I'm not an aiohttp user but noticed this when I was trying out the framework and wanted to make sure developers were aware of it.  I do not consider this a feature, but more of a security vulnerability and [nobody else seems to know they will need to enable autoescape when using this project](https://github.com/search?utf8=%E2%9C%93&q=aiohttp_jinja2.setup&type=Code
)

Everybody is busy and in this case both our time has been wasted with this back and forth. I will try again but please consider how much effort you put in typing the above versus actually fixing this issue.